OsmAnd/src/net/osmand/plus/views/OsmandMapTileView.java

```diff
@@ -1081,8 +1081,8 @@ private void refreshMapInternal(@NonNull DrawSettings drawSettings) {
 			return;
 		}
 		PointF ratio = mapViewTrackingUtilities.getMapDisplayPositionManager().getMapRatio();
-		int cy = (int) (ratio.y * view.getHeight());
-		int cx = (int) (ratio.x * view.getWidth());
+		int cy = Math.round(ratio.y * view.getHeight());
```

Comment:
Do we need all those Math.round operations? They caused this bug.
https://github.com/osmandapp/OsmAnd/issues/15034#issuecomment-3298035234